src/atoms/Flex/Flex.web.js

```diff
@@ -1,3 +1,76 @@
-const Flex = () => null;
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Flex = styled(
+  ({
+    alignContent,
+    alignItems,
+    alignSelf,
+    display,
+    flex,
+    flexBasis,
+    flexDirection,
+    flexGrow,
+    flexShrink,
+    flexWrap,
+    justifyContent,
+    order,
+    children,
+    style,
```

Comment:
remove this `style` from here. don't explicitly extract it. Native needs it but web doesn't

Reply:
removed